docs/source/components.md

```diff
@@ -0,0 +1,319 @@
+        Term("delier", attr="NORM"),
+        Term("DOS", attr="TEXT"),
+        [
+             {"NORM": {"IN": ["zag", "ziet", "hoort", "hoorde", "ruikt", "rook"]}},
+             {"OP": "?"},
+             {"OP": "?"},
+             {"OP": "?"},
+             {"NORM": {"FUZZY1": "dingen"}},
+             {"OP": "?"},
+             {"NORM": "die"},
+             {"NORM": "er"},
+             {"OP": "?"},
+             {"NORM": "niet"},
+             {"OP": "?"},
+             {"NORM": {"IN": ["zijn", "waren"]}}
+        ],
+    ]
+}
+```
+
+#### Concept dictionary from external source
+
+External lists of concepts (e.g. from a medical thesaurus such as `UMLS`) can also be loaded directly from `csv` through the `create_concept_dict` function. Your `csv` should contain a combination of concept and phrase on each line, with optional columns to configure the `Term`-options described above (e.g. `attribute`, `proximity`, `fuzzy`). You may present the columns in any order, but make sure the names match the `Term` attributes. Any other columns are ignored. For example:
+
+| **concept** | **phrase** | **attr** | **proximity** | **fuzzy** | **fuzzy_min_len** | **pseudo** | **comment** |
+|--|--|--|--|--|--|--|--|
+| prematuriteit | prematuriteit | | | | | | some comment |
+| prematuriteit | <p3 | | 1 | 1 | 2 | | |
+| hypotensie | hypotensie | | | | | | |
+| hypotensie | bd verlaagd | | 1 | | | | |
+| veneus_infarct | veneus infarct | | | | | | |
+| veneus_infarct | VI | TEXT | | | | | |
+
+Will result in the following concept dictionary:
+
+```python
+{
+    "prematuriteit": [
+        "prematuriteit",
+        Term("<p3", proximity=1, fuzzy=1, fuzzy_min_len=2),
+    ],
+    "hypotensie": [
+        "hypotensie",
+        Term("bd verlaagd", proximity=1)
+    ],
+    "veneus_infarct": [
+        "veneus infarct",
+        Term("VI", attr="TEXT")
+    ]
+}
+```
+
+## Qualification
+
+### `clinlp_context_algorithm`
+
+| property | value |
+| --- | --- |
+| name | `clinlp_context_algorithm` |
+| class | [clinlp.ie.qualifier.context_algorithm.ContextAlgorithm](clinlp.ie.qualifier.context_algorithm.ContextAlgorithm) |
+| example | `nlp.add_pipe('clinlp_context_algorithm')` |
+| requires | `doc.sents`, `doc.spans['ents']` |
+| assigns | `span._.qualifiers` |
+| config options | `phrase_matcher_attr = "TEXT"` <br /> `load_rules = True` <br /> `rules = "src/clinlp/resources/context_rules.json"` |
+
+The rule-based [Context Algorithm](https://doi.org/10.1016%2Fj.jbi.2009.05.002) is fairly accurate, and quite transparent and fast. A set of rules, that checks for `Presence`, `Temporality`, and `Experiencer`, is loaded by default:
+
+```python
+nlp.add_pipe("clinlp_context_algorithm", config={"phrase_matcher_attr": "NORM"})
+```
+
+A custom set of rules, including different types of qualifiers, can easily be defined. See [`src/clinlp/resources/context_rules.json`](../../src/clinlp/resources/context_rules.json) for an example, and load it as follows:
```

Comment:
Moet dit een download zijn, of kan het ook verwijzen naar het bestand op Github o.i.d., zodat je het kan inzien maar dus niet hoeft te downloaden.